src/components/plots/CategoricalEmbeddingPlot.jsx

```diff
@@ -24,7 +24,7 @@ const CategoricalEmbeddingPlot = (props) => {
     (state) => state.experimentSettings.processing?.configureEmbedding?.embeddingSettings,
   );
   const { data: embeddingData, loading, error } = useSelector(
-    (state) => state.embeddings[embeddingSettings.method],
+    (state) => (embeddingSettings ? state.embeddings[embeddingSettings.method] : {}),
```

Comment:
These are the same changes that @StefanBabukov did here: https://github.com/biomage-ltd/ui/pull/215/files

Reply:
Good you noticed! Not only that, but his version is better because he use a better guard for one of the `useEffect()` calls! Will remove.